src/Pages/Services/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { Container, Header, ServicesList, ServiceItem, ServiceImage, ServiceName, ServiceDetails, ServiceDescription, SelectedServiceImage } from './style';
+import { services, Service } from './constant';
+
+const ServicesPage: React.FC = () => {
+const [selectedService, setSelectedService] = useState<Service | null>(null);
+const serviceDetailsRef = useRef<HTMLDivElement>(null);
+
+useEffect(() => {
+if (selectedService && serviceDetailsRef.current) {
+window.scrollTo({
+top: serviceDetailsRef.current.offsetTop,
+behavior: 'smooth'
+});
+}
+}, [selectedService]);
+
+const handleServiceSelect = (service: Service) => {
+setSelectedService(service);
+};
+
+return (
+<Container>
+<Header>Our Services</Header>
+<ServicesList>
+{services.map(service => (
+<ServiceItem key={service.name} onClick={() => handleServiceSelect(service)}>
+<ServiceName>{service.name}</ServiceName>
```

Comment:
What is this? There is no formatting? I have already said that if the files are not formated, it should not allow to push PR. Who is working on that check?

Reply:
I had some problem in installation of prettier ,I resolved it and update the files.